connect-examples/v2/php_checkout/README.md

```diff
@@ -0,0 +1,28 @@
+Square Checkout Demo
+=========================
+
+This is a simple example application that utilizes Square's Checkout API.
+
+It takes a single payment, declared by the user, and creates an order to use in the Checkout API.
+
+To get it running:
+
+* Clone/download to your local computer.
+* Rename `.envsample` to `.env` and fill in the values.
```

Comment:
I can do this, but (as I mentioned to Shao above), I'm afraid of accidentally pushing credentials. If I were testing and put my credentials in .env, and don't ignore it, then I could push it and now everyone has my credentials. This would be the case for anyone using it. If this is the preferred method, then I will go ahead and do it, though.

Reply:
Replied in Shao's comment thread.